packages/ui-extensions/docs/surfaces/admin/staticPages/admin-extensions.doc.ts

```diff
@@ -0,0 +1,179 @@
+      type: 'Generic',
+      anchorLink: 'actionExtensions',
+      title: 'Overview',
+      image: 'action-extension-example.gif',
+      sectionContent: 'Extend the Shopify Admin with App Extensions.',
+      sectionNotice: [
+        {
+          title: 'Developer Preview',
+          sectionContent:
+            'Please not that Admin Action and Block Extensions are currently in a Developer Preview. You may on see them while developing locally on a development store.',
+          type: 'warning',
+        },
+      ],
+      sectionCard: [
+        {
+          name: 'Getting Started',
+          subtitle: 'Get started building your first Admin Extension.',
+          url: '#gettingStarted',
+          type: 'tutorial',
+        },
+        {
+          name: 'Components',
+          subtitle: 'See all available components.',
+          url: 'admin-extensions/components',
+          type: 'blocks',
+        },
+        {
+          name: 'Extension Targets',
+          subtitle: 'View a list of available extension targets.',
+          url: 'apis/extension-points',
+          type: 'app',
+        },
+        {
+          name: 'Direct API access',
+          subtitle:
+            "You no longer need a backend proxy to talk to Shopify's GraphQL Admin API.",
+          url: '#directAPIAccess',
+          type: 'tool',
+        },
+        {
+          name: 'Custom Protocols',
+          subtitle:
+            "We've made it easier to construct URLs to navigate to common localtions.",
+          url: '#directAPIAccess',
+          type: 'gear',
+        },
+      ],
+    },
+    {
+      type: 'Generic',
+      anchorLink: 'gettingStarted',
+      sectionContent:
+        'To get started, create a new app and cd into the app directory. Then generate an extension.',
+      title: 'Getting Started',
+      codeblock: {
+        title: 'Generate an extension',
+        tabs: [
+          {
+            code: './examples/getting-started.ts',
+            language: 'bash',
+            title: 'CLI',
+          },
+        ],
+      },
+    },
+    {
+      type: 'Generic',
+      title: 'Direct API access',
+      sectionContent:
+        "You can now make requests to shopify directly from your extension. You can simply use fetch! The access scopes you have in your extension will be the same as the access scopes you have in your app. If you need to make a request to a resource that requires a different access scope, you will update your app's scopes.",
+      anchorLink: 'directAPIAccess',
+      codeblock: {
+        title: 'Query Shopify data',
+        tabs: [
+          {
+            code: './examples/direct-api.tsx',
+            language: 'tsx',
+            title: 'Get Product Data',
+          },
+        ],
+      },
+    },
+    {
+      type: 'GenericAccordion',
+      title: 'Custom Protocols',
+      sectionContent:
+        "We've added some custom protocols to make it easier to construct URLs to navigate to common localtions.",
```

Comment:
Update to actionable language and less self referential.

`Custom protocols make it easier to navigate to common locations, and construct URLs.`

Reply:
```suggestion
        "Custom protocols make it easier to navigate to common locations, and construct URLs.",
```